docs/section19/19-19.md

```diff
@@ -0,0 +1,96 @@
+    - Create dictionary of hvac systems, zones, and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in target_system_types):`         
+            - Get the list of zones associated with the hvac system: `zone_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac.id]["ZONE_LIST"]`  
+            - Reset supply cfm to zero: `supply_cfm_b = 0`  
+            - Reset total_fan_power to zero: `total_fan_power = 0`   
+            - Reset fan_power_W_CFM to zero: `fan_power_W_CFM = 0`    
+            - Reset total exhaust cfm: `total_exhaust_cfm = 0`       
+            - Reset total relief cfm: `total_relief_cfm = 0`       
+            - Get the fan system object:  `fan_sys_b = hvac.fan_system`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan powers in kW: `fan_sys_kW = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"])`  
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])` 
+                    
+            - Get the hvac system total supply fan kW, need to get the first element in the list of fan powers:`hvac_sys_total_supply_fan_power_b = fan_sys_kW[0]` 
+            - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power_b = fan_sys_kW[1]` 
+            - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power_b = fan_sys_kW[2]` 
+            - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power_b = fan_sys_kW[3]`     
+
+            - Calculate the total modeled fan power for the hvac system in the baseline: `total_fan_power = hvac_sys_total_supply_fan_power_b + hvac_sys_total_return_fan_power_b + hvac_sys_total_exhaust_fan_power_b + hvac_sys_total_relief_fan_power_b`  
```

Comment:
Question: How is like 41 different from line 48? Isnt the function getting thee total kw? and then line 48 is summing up the individual kw to come to the same answer?

Reply:
This does not include terminal fans. Sounds like these should be added even though we are not expecting them for these system types.

Reply:
@supriyagoel good Q, 41 is not a total, it creates a list with kW as values for supply, return, exhaust, and relief fans. We need it is total and separate so I think we need this line to create the list rather than just summing them right off the bat.

Reply:
Resolved on call. 2/16/2023